lib/Sema/PreCheckExpr.cpp

```diff
@@ -2119,8 +2119,7 @@ Expr *PreCheckExpression::simplifyTypeConstructionWithLiteralArg(Expr *E) {
       return nullptr;
   }
 
-  SmallVector<ProtocolConformance *, 2> conformances;
-  return castTy->getAnyNominal()->lookupConformance(protocol, conformances)
+  return DC->getParentModule()->lookupConformance(castTy, protocol)
```

Comment:
Would it still work for a local type?

Reply:
Or perhaps that’s too marginal a case anyway.

Reply:
ModuleDecl::lookupConformance() works for all types, in fact it ignores `this`.